src/components/Media/_media.scss

```diff
@@ -29,21 +29,14 @@
 
 .cc-media__caption {
   align-self: start;
-  // D7 override
-  border: 0;
-  border-left: 1px solid var(--color-grey-70);
-  color: var(--color-grey-70);
-  font-size: var(--body-xxs);
-  line-height: var(--body-line-height);
   margin-top: calc(2 * var(--space-unit));
-  padding: calc(0.25 * var(--space-unit)) 0 calc(0.25 * var(--space-unit)) calc(2 * var(--space-unit));
 
   a {
     @include animated-link;
   }
 }
 
-.cc-media__caption--below-image {
+.cc-media--caption--below-image {
```

Comment:
`cc-media--caption-below-image` (we should remove an extra hyphen).

Reply:
Doh!

Reply:
Fixed